src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/integration/RegistrationEdgeCasesTest.kt

```diff
@@ -46,6 +49,7 @@ class RegistrationEdgeCasesTest : IntegrationTestBase() {
   @Test
   fun `exclude historic registrations from tier calculation`() {
     val crn = "X445599"
+    setupTierToDeliusNoAssess(crn)
```

Comment:
is Assess here short for Assessment? could the method be updated to have the full word?